docs/contribute/contribution_guidelines.rst

```diff
@@ -70,6 +70,15 @@ The testing for functions and classes should at least contain tests for
 
 In case of deprecations, it must also be tested if the deprecated feature is properly removed (see `tests/test_deprecations.py` for examples).
 
+Testing plotting functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The above also holds for plotting functions, however, in this case it is required that the plots generate baseline and test plots. Baseline plots are visually inspected when they are first created and serve as a reference. Test plots are created each time the test are ran. If a pull request changes a plot function, the test plots are visually compared against the baseline plots.
+
+``pyfar.testing.plot_utils`` contains the function ``create_figure`` which is best practice for creating a figure in a way that produces almost identical results across operating systems. The function ``save_and_compare`` is best practice for saving and comparing test plots automatically. However, the automatic comparison is usually disabled. It often fails because the plots generated by Matplotlib depend on the operating system. Different operating systems create offsets of a few pixels resulting in large differences between test and baseline plots.
```

Comment:
it might make sence to add the pyfar.testing module to the documentation, since we need it in other packages. then we can also directly link to it here. we can also create a link in the module to this page.

Reply:
I agree that it would be good to document this. I think the best way to do this would be to add pyfar.testing to a separate section of the official docs and then link it here. I would suggest to do this in a separate pull.